sschipke/palette-producer
# Ss testing

## What's this PR do?
- adds snapshots tests
- finishes tests for apiCalls
### Where should the reviewer start?
- all the test fils
### How should this be manually tested?
- run `npm test` or `npm test --  --coverage --watchAll=false` to see test coverage
### What are the relevant tickets?
- testing
#### Screenshots (if appropriate)
